{"name": "test_qlogin_normal", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <test_login.TestUserLogin object at 0x0000028A8444B250>\n\n    def test_qlogin_normal(self):\n        case_data = get_test_data(self.data_list, \"test_qlogin_normal1\")\n        if not case_data:  # 有可能为None\n            logging.error(\"用例数据不存在\")\n    \n>       url = case_data.get('url')\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntest_login.py:19: AttributeError"}, "attachments": [{"name": "log", "source": "9c0ce260-24c0-4d2a-8b16-32030a48b0f9-attachment.txt", "type": "text/plain"}], "start": 1614930743238, "stop": 1614930743239, "uuid": "1ff188ee-c6a1-480e-ba41-21a4b8e0f83f", "historyId": "415ce8af72144b8e2e2a2835b3d3e60d", "testCaseId": "0210d8c0eab07c4329b10bb4c3919f81", "fullName": "test_login.TestUserLogin#test_qlogin_normal", "labels": [{"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "liailian"}, {"name": "thread", "value": "22724-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}